openscap_report/scap_results_parser/oval_and_cpe_tree_builder.py

```diff
@@ -98,13 +134,29 @@ def _remove_double_cpe_requirement(rule, group_platforms):
             if platform in group_platforms:
                 group_platforms.remove(platform)
 
+    def get_oval_definition(self, rule):
+        report = self.reports_with_oval_definitions.get(rule.oval_reference, None)
+        if report is not None:
+            return report.get(rule.oval_definition_id)
+        oval_def = None
+        for report in self.reports_with_oval_definitions.values():
+            if oval_def is not None and rule.oval_definition_id in report:
+                logging.warning(
+                    ("The given input contains the duplicate results of"
+                     " the OVAL definition (\"%s\")."),
```

Comment:
Make the way you split strings consistent: space either goes to the end or to the start of the line.